Assets/Scripts/ROS Scripts/ROS Connections v2/Sensors/MeshSensor_ROSSensorConnection.cs

```diff
@@ -20,7 +20,7 @@ public class MeshSensor_ROSSensorConnection : MonoBehaviour, ROSTopicSubscriber,
     private ROSBridgeWebSocketConnection ros = null;
     public string client_id;
     private Thread rosMsgThread;
-    private List<string> sensorSubscriberTopics = new List<string>();
+    private Dictionary<string, bool> sensorSubscriberTopicsDict = new Dictionary<string, bool>();
```

Comment:
Can we add the dict to the ROSTopicSubscriber?

Reply:
What do you mean?